tests/providers/vertica/hooks/test_vertica.py

```diff
@@ -47,6 +47,57 @@ def test_get_conn(self, mock_connect):
             host="host", port=5433, database="vertica", user="login", password="password"
         )
 
+    @patch("airflow.providers.vertica.hooks.vertica.connect")
+    def test_get_conn_extra_parameters_no_cast(self, mock_connect):
+        extra_dict = self.connection.extra_dejson
```

Comment:
what is the purpose of this test? Can you please add docstring here. It will be helpful for everyone